tests/test_flow/fakes.py

```diff
@@ -164,14 +148,13 @@ def _run_aip_job(self, body):
         # change location of disk cache for the AIP job.
         self._modify_disk_cache_location(input_uri)
 
-        # AIP execution does not send back logs to local bionic instance. Hence,
-        # all logs are filtered here.
-        with temporarily_filter_all_logs():
-            try:
-                run_aip(input_uri, self._gcs_fs)
-                return "SUCCEEDED", None
-            except Exception as e:
-                return "FAILED", str(e)
+        try:
+            # TODO: filter out logs. AIP execution does not send back logs to
```

Comment:
Why this change?

Reply:
It may not be working correctly as expected. In addition, the Syncmanager Barrier was also not working correctly; all the entities in `test_parallel_fail`, including the ones that do not throw exceptions, were failing. Hence, I have changed the fake AIP to use a process executor instead of a thread executor.